docs/examples/barcode-image/index.html

```diff
@@ -26,7 +26,7 @@ <h1 class="title">Scan barcode from Image</h1>
 				</div>
 
 				<div>
-					<img id="img" src="../img/2of5-interleaved_806002000537753383.png" style="border: 1px solid gray"></img>
+					<img id="img" src="../../../src/test/core/resources/blackbox/ean13-1/1.png" style="border: 1px solid gray"></img>
```

Comment:
Can we make an example for EAN 13 without removing the ITF one? After that I think we're ready to merge.

Reply:
Agree

Reply:
I had removed IFT without knowing it because the link was broken.
I did a bit of refactoring which I hope you like.
Now every barcode has its "test" section.

Reply:
A fact: we gotta work on that examples. They all deserve to have a demo. 😅 Getting them ready is a v1.0.0 goal.